public/swagger.html

```diff
@@ -0,0 +1,19 @@
+<!-- HTML for static distribution bundle build -->
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>CMS - CoW Protocol</title>
+    <link rel="stylesheet" type="text/css" href="./swagger-ui-v5.15.2/swagger-ui.css" />
+    <link rel="stylesheet" type="text/css" href="./swagger-ui-v5.15.2/index.css" />
+    <link rel="icon" type="image/png" href="./swagger-ui-v5.15.2/favicon-32x32.png" sizes="32x32" />
+    <link rel="icon" type="image/png" href="./swagger-ui-v5.15.2/favicon-16x16.png" sizes="16x16" />
+  </head>
+
+  <body>
+    <div id="swagger-ui"></div>
+    <script src="./swagger-ui-v5.15.2/swagger-ui-bundle.js" charset="UTF-8"> </script>
+    <script src="./swagger-ui-v5.15.2/swagger-ui-standalone-preset.js" charset="UTF-8"> </script>
+    <script src="./swagger-initializer.js" charset="UTF-8"> </script>
```

Comment:
I assume this needs to be part of the bundle and doesn't work if we include the direct dependency like you did on cow.fi? 

https://github.com/cowprotocol/cow-fi/pull/202/files#diff-030d043c863664a64c233ce8eb0a91a3123a16470663d60c39b58fe0b84aef50R9-R14

Reply:
Yes, that didn't work in this case because of some security headers, so I decided to get the standalone app from the repo: https://github.com/swagger-api/swagger-ui/releases